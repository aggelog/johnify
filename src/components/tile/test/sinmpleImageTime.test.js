import React from 'react';
import SimpleImageTile from '../simpleImageTile';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<SimpleImageTile url="www.google.com" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});