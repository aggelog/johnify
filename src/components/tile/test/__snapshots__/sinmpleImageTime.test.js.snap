// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "height": 100,
      "margin": "auto",
      "maxWidth": "100%",
      "width": 100,
    }
  }
>
  <img
    alt="..."
    src="www.google.com"
    style={
      Object {
        "maxWidth": "100%",
      }
    }
  />
</div>
`;
